puppet/modules/kubernetes/templates/aws-iam-authenticator-init.service.erb

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Initialise heptio authenticator (AWS IAM authenticator)
+Documentation=https://github.com/kubernetes-sigs/aws-iam-authenticator
+After=network.target
+After=docker.service
+Requires=docker.service
+Before=kube-apiserver.service
+<%= scope.function_template(['kubernetes/_systemd_unit.erb']) %>
+[Service]
+Type=oneshot
+ExecStartPre=/bin/docker run \
```

Comment:
Is there a chance we can get the binary instead and just run it without docker?

This won't be cleaning up the container, will restart with docker restarts ...

Wdyt @JoshVanL @seanturner83